src/mapgen_valleys.cpp

```diff
@@ -418,9 +332,8 @@ void MapgenValleys::calculateNoise()
 	}
 
 	for (s32 index = 0; index < csize.X * csize.Z; index++) {
```

Comment:
Yeah but the particular case you pointed out I didn't *add* any of it, just renamed other identifiers that happen to be in the same expression as the shadowed variable.